src/hooks/useAuth.js

```diff
@@ -1,19 +1,26 @@
 import { useEffect } from 'react';
-import { useSearchParams } from 'next/navigation';
+import { useSearchParams, useRouter } from 'next/navigation';
 import axios from 'axios';
 import { useGlobalContext } from 'app/(context)';
+import { NOW_PLAYING } from 'public/constants/pathNames';
 
 export default function useAuth() {
   const searchParams = useSearchParams();
+  const router = useRouter();
   const { credentials, setCredentials } = useGlobalContext();
   useEffect(() => {
+    if (credentials.accessToken) {
+      router.push('/now-playing');
+    }
+
```

Comment:
Should use the constant value here as well for consistency.
```suggestion
  useEffect(() => {
    if (credentials.accessToken) {
      router.push(NOW_PLAYING);
    }
```